tensorboard/data_compat_test.py

```diff
@@ -61,22 +61,24 @@ def _assert_noop(self, value):
     self.assertEqual(original_pbtxt, value.SerializeToString())
 
   def test_scalar(self):
-    old_op = tf.compat.v1.summary.scalar('important_constants', tf.constant(0x5f3759df))
-    old_value = self._value_from_op(old_op)
-    assert old_value.HasField('simple_value'), old_value
-    new_value = data_compat.migrate_value(old_value)
-
-    self.assertEqual('important_constants', new_value.tag)
-    expected_metadata = scalar_metadata.create_summary_metadata(
-        display_name='important_constants',
-        description='')
-    self.assertEqual(expected_metadata, new_value.metadata)
-    self.assertTrue(new_value.HasField('tensor'))
-    data = tensor_util.make_ndarray(new_value.tensor)
-    self.assertEqual((), data.shape)
-    low_precision_value = np.array(0x5f3759df).astype('float32').item()
-    self.assertEqual(low_precision_value, data.item())
-
+    with tf.compat.v1.Session():
```

Comment:
I'm a bit confused about using a session context manager here but not actually including `as sess` - does anything actually use this session in the code below?

It seems like perhaps this is being used just as a way to enable a default graph, but then I think `with tf.compat.v1.Graph.as_default()` would be clearer?

Also, it would be great to scope these contexts to just where they're really needed.  In this case I suspect it's just the first couple lines; the rest of these aren't executing TF code.

Reply:
It seems like we need then fix the problem elsewhere too.
Since I am more interested in enabling v2 tests right now, I will let future commits fix things like `tensorboard.compat.tf2` (or is that version correct as is? Even considering the env variable?)

Reply:
Done.